notation/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "notation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
This license was the default while running `npm init`. Shall we use [APACHE LICENSE, VERSION 2.0](https://www.apache.org/licenses/LICENSE-2.0) instead?

Reply:
Updated to be `MIT` license because all the [azure-pipelines-tasks](https://github.com/microsoft/azure-pipelines-task-lib) repo is MIT and all the tasks should follow he MIT license.